.contrib/Harvesters/WoWTools Harvester/Harvester.py

```diff
@@ -1,10 +1,12 @@
-### Tool Harvesters and Generating Missing with name
-import os
-import re
+# Tool Harvesters and Generating Missing with name
 import csv
+import re
+from enum import Enum, auto
+from pathlib import Path
+
 import requests
-import time
-from bs4 import BeautifulSoup
+
+DATAS_FOLDER = Path("..", "..", "Parser", "DATAS")
 
 profession_dict = {
```

Comment:
This is not used anywhere.

Reply:
Use a throttle or time.sleep() only for getting the raw files.